client/src/styles/AccountSettingsPage.scss

```diff
@@ -3,20 +3,37 @@
 @import "_callout.scss";
 
 .AccountSettingsPage {
-  h4 {
-    text-decoration: none;
-    text-align: left !important;
-    font-weight: 700;
-    padding-top: 0.37rem;
-    margin: 3.5rem 1rem 1.63rem;
+  .page-container {
+    padding-top: 1.5rem;
   }
 
-  h4.settings-section {
+  h1 {
+    font-size: 24px;
     font-weight: 400;
```

Comment:
I see in figma this is 400, but it's 500 on standalone pages, should this be the same?
```suggestion
    font-weight: 500;
```

Reply:
having so much trouble linking the figma frame but the email verification standalone pages also have font-weight 400. I think it may be best to match it to whatever figma has for now, even though I personally feel like the weights should be the same 

<img width="1077" alt="Screenshot 2024-04-05 at 1 03 25 PM" src="https://github.com/JustFixNYC/who-owns-what/assets/17574626/5d7fc16e-f44e-4449-affb-b882b3321f32">

Reply:
hmm yeah that makes sense, I'll add a ticket for this

Reply:
https://app.shortcut.com/justfixnyc/story/14346/consistent-font-weight-for-headers-on-standalone-and-account-settings-pages